doc/sphinxext/gallery_generator.py

```diff
@@ -92,12 +93,27 @@
       :shadow: none
       :class-card: example-gallery
 
+      .. div:: example-img-plot-overlay
+
+         {overlay}
+
       .. image:: ./matplotlib/{pngfilename}
 
       +++
       {title}
 """
-#:img-top: ./matplotlib/{pngfilename}
+
+CATEGORIES = [
```

Comment:
These categories are fine. We can change them ourselves now that you've created the framework

Reply:
Awesome. I'll just add the rest of what Oriol recommended. I think I'll do a scan of each chart as I transition them into a category + `arviz-doc` style. That way it's easier to have critique.

Reply:
Thank you